mne/source_space.py

```diff
@@ -1432,11 +1433,17 @@ def setup_volume_source_space(subject, fname=None, pos=5.0, mri=None,
         if mri is None:
             raise RuntimeError('"mri" must be provided if "volume_label" is '
                                'not None')
+        if not isinstance(volume_label, list):
+                volume_label = [volume_label]
+
         # Check that volume label is found in .mgz file
-        volume_labels = get_volume_labels_from_aseg(mri)
-        if volume_label not in volume_labels:
-            raise ValueError('Volume %s not found in file %s. Double check '
-                             'freesurfer lookup table.' % (volume_label, mri))
+        volume_labels, _ = get_volume_labels_from_aseg(mri)
```

Comment:
yes, but I changed all modules calling this function in this way:
`label_names, __ = get_volume_labels_from_aseg(fname_aseg)`

but now I changed again following your suggestions to add the parameter
`return_colors`